bench/worker/worker_bench.rb

```diff
@@ -55,6 +56,15 @@ def dup_resque_redis
   end
 
   def run
+    n_reports = [*@nreports.strip.split(',')].map(&:to_i)
```

Comment:
I'm not sure why do we need `[*...]`. Also perhaps more robust to strip each value in case user enters some spaces in between, e.g. `100, 1000 , 10000`
```suggestion
    n_reports = @nreports.split(',').map(&:strip).map(&:to_i)
```